deploy/olm-catalog/ibm-platform-api-operator/3.6.0/ibm-platform-api-operator.v3.6.0.clusterserviceversion.yaml

```diff
@@ -95,9 +95,11 @@ metadata:
     certified: "false"
     containerImage: quay.io/opencloudio/ibm-platform-api-operator:latest
     createdAt: "2020-03-25"
-    description: The PlatformAPI operator provides a simple Kubernetes CRD-Based API
-      to manage the lifecycle of PlatformAPI. With this operator, you can simply deploy
-      and upgrade platform-api.
+    description: "#IBM Platform API Operator\n> **Important:** Do not install this operator directly. Only install this operator using the IBM Common Services Operator. For more information about installing this operator and other Common Services operators, see [Installer documentation](http://ibm.biz/cpcs_opinstall). If you are using this operator as part of an IBM Cloud Pak, see the documentation for that IBM Cloud Pak to learn more about how to install and use the operator service. For more information about IBM Cloud Paks, see [IBM Cloud Paks that use Common Services](http://ibm.biz/cpcs_cloudpaks).
```

Comment:
There are two descriptions. I think the first one is for the side panel, the second part is for the detailed description on the main panel. It's the second description that should be updated

Reply:
are you talking about the description field on line 111?